stories/assembly-aap-aws-intro.adoc

```diff
@@ -18,5 +18,9 @@ The following {PlatformName} components are available on {AAPonAWS}:
 * {ExecEnvNameStart}
 * Ansible content tools, including access to {InsightsName}
 
+[NOTE]
+=====
+{AutomationMeshStart} is not available on {AAPonAWS} at this time and is planned for a later release.
```

Comment:
@ianf77 
Would it be better not to mention that Mesh is planned for a later release?

Reply:
I can do.